app/models/job_execution.rb

```diff
@@ -125,6 +127,7 @@ def run!
     finish
     ActiveRecord::Base.clear_active_connections!
   end
+  add_transaction_tracer :run!, category: :task, params: '{ job_id: id, project: job.project.try(:name), reference: reference }'
```

Comment:
Nope.  Single-quoted so that it will be interpolated when `run!` is called, rather than when `add_transaction_tracer` is called.  It's described in the [New Relic rubydocs](http://www.rubydoc.info/github/newrelic/rpm/NewRelic/Agent/Instrumentation/ControllerInstrumentation/ClassMethods#add_transaction_tracer-instance_method)